workspaces/redhat-resource-optimization/README.md

```diff
@@ -1,8 +1,140 @@
+   );
+   ```
+
+1. Add the `ResourceOptimizationPage` extension to your `App.tsx` routes
+
+   ```ts
+   // packages/app/src/App.tsx
+
+   import { ResourceOptimizationPage } from '@backstage-community/plugin-redhat-resource-optimization';
+
+   <FlatRoutes>
+     ...
+     <Route
+       path="/redhat-resource-optimization"
+       element={<ResourceOptimizationPage />}
+     />
+     ...
+   </FlatRoutes>;
+   ```
+
+1. Add a link to the Resource Optimization page in the side bar
+
+   ```diff
+   // packages/app/src/components/Root/Root.tsx
+
+   + import { ResourceOptimizationIconOutlined } from '@backstage-community/plugin-redhat-resource-optimization';
+
+   export const Root = ({ children }: PropsWithChildren<{}>) => (
+     <SidebarPage>
+       <Sidebar>
+         <SidebarLogo />
+         <SidebarGroup label="Search" icon={<SearchIcon />} to="/search">
+           <SidebarSearchModal />
+         </SidebarGroup>
+         <SidebarDivider />
+         <SidebarGroup label="Menu" icon={<MenuIcon />}>
+           {/* Global nav, not org-specific */}
+           <SidebarItem icon={HomeIcon} to="catalog" text="Home" />
+           <SidebarItem icon={ExtensionIcon} to="api-docs" text="APIs" />
+           <SidebarItem icon={LibraryBooks} to="docs" text="Docs" />
+           <SidebarItem icon={CreateComponentIcon} to="create" text="Create..." />
+           {/* End global nav */}
+           <SidebarDivider />
+           <SidebarScrollWrapper>
+             <SidebarItem icon={MapIcon} to="tech-radar" text="Tech Radar" />
+           </SidebarScrollWrapper>
+   +       <SidebarItem
+   +         icon={ResourceOptimizationIconOutlined}
+   +         to="/redhat-resource-optimization"
+   +         text="Optimizations"
+   +       />
+         </SidebarGroup>
+         <SidebarSpace />
+         <SidebarDivider />
+         <SidebarGroup
+           label="Settings"
+           icon={<UserSettingsSignInAvatar />}
+           to="/settings"
+         >
+           <SidebarSettings />
+         </SidebarGroup>
+       </Sidebar>
+       {children}
+     </SidebarPage>
+   );
+   ```
+
+#### Option 2: Dynamic plugin
+
+[Follow this link](./docs/dynamic-plugin.md) for installing plugin as Dynamic Plugin
+
+### Dependency on Orchestrator plugin and Workflow details
+
+The Resource Optimization plugin is dependent on [Orchestrator plugin](https://www.rhdhorchestrator.io/main/docs/) to run the workflow for applying the recommendation. Make sure you have installed the [Orchestrator plugin](https://www.rhdhorchestrator.io/main/docs/) by following one of these options depending on your environment:
+
+- [Install as a static plugin for local development](https://github.com/redhat-developer/rhdh-plugins/tree/main/workspaces/orchestrator#install-as-a-static-plugin)
+- [Install Orchestrator plugin on exisiting RHDH instance](https://www.rhdhorchestrator.io/main/docs/installation/)
```

Comment:
The orchestrator plugin can either be installed on an existing RHDH instance or the orchestrator operator can install RHDH and the orchestrator at the same time.

Reply:
@chadcrum okay thanks for sharing the same @chadcrum I have mentioned the link to orchestrator plugin docs, do you think anything else will be needed here apart from that?

Reply:
@pkliczewski @chadcrum thank you for your inputs, I have updated the README by including both the links in [this commit](https://github.com/jkilzi/backstage-community-plugins/pull/43/commits/c940833f3d9922f67a5797e6aeedfe94f9b3a292). Hence marking this as resolve.